### PR TITLE
Changes that needed to be committed from PR 835

### PR DESCRIPTION
**Pull request by kwiedenhaupt**
1. Changing the enum to inProfile and then changing associated index files.

### CODE REVIEW

File: components/coding/GoalsSectionComponent.tsx [hunk 1]
```diff
@@ -80,14 +80,12 @@ export default function GoalsSection({
         })}
       </div>
     </ExpandableContainer>
-  ) : userGoals.length <= 0 &&
-    componentUsageType === GoalsComponentSection.PROFILE ? (
+  ) : userGoals.length <= 0 && inProfile === true ? (
```
Comment (iamnotreddy) [inline components/coding/GoalsSectionComponent.tsx:83]: tip: when referencing a boolean, the default is always true so you can just write "inProfile" - does the same thing as your code above

File: pages/profile/v2/index.tsx [hunk 2]
```diff
@@ -54,7 +52,7 @@ export default function Profile(props) {
 
       <div className="grid grid-cols-1">
         <GoalsSectionComponent
-          componentUsageType={GoalsComponentSection.PROFILE}
+          inProfile={true}
```
Comment (iamnotreddy) [inline pages/profile/v2/index.tsx:55]: same thing here, you could just write inProfile

File: pages/goals/index.tsx [hunk 3]
```diff
@@ -83,7 +81,7 @@ export default function Goals(props) {
               <GoalsSectionComponent
                 userGoals={section.userGoals}
                 sectionName={section.sectionName}
-                componentUsageType={GoalsComponentSection.GOALS}
+                inProfile={false}
```
Comment (iamnotreddy) [inline pages/goals/index.tsx:84]: over here, you could write !inProfile for shorthand to pass the value as false into the component

**Review by iamnotreddy — APPROVED** [review]
approved!

you don't have to change this but one piece of feedback - I think the use of ternary operators in your rendering logic is a bit confusing, especially since you're evaluating more than two conditions... in these kinds of cases, i think it's better for readability to use an if statement

**Comment by kwiedenhaupt** [comment]
Screen Shot 2022-12-29 at 4 37 34 PM
Brand new approach focused on perfecting the query and then dispatching data, and it seems to be working!

**Pull request merged**